vectors/cryptography_vectors/asymmetric/RSA/FIPS_186-2/Readme.txt

```diff
@@ -4,6 +4,6 @@ Example test files for FIPS 186-2 RSA
 
 2. The file SigGenRSA_186-2.txt contains values for X9.31RSA signature generation with the additional value d added to the file for testing purposes.
 
-3.  The file SigGen15_186-2.txt contains values for RSA PKCS#1 Ver 1.5 signature geenration with the additional value d added to the file for testing purposes.
+3.  The file SigGen15_186-2.txt contains values for RSA PKCS#1 Ver 1.5 signature generation with the additional value d added to the file for testing purposes.

```

Comment:
This file is an exact replica of a vector from NIST and shouldn't be modified.

Reply:
Fixed.